Document Linux distribution requirements (#5333)
flutter/website: src/desktop.md

#### Linux

The executable can be found in your project under `build/linux/<build mode>/bundle/`. Alongside your 

> executable binary in the `bundle` directory there are two directories, `lib` which contains required `.so`

Reviewer: Nit: s/,/:/

Author: Refactored the two points into a list, following the new `:` in the text.

> library files, and the `data` directory that contains data assets that your application relies on.

Reviewer: Nit: s/that/which/ for parallel structure.

Author: Re-worded, but I'm unsure if I like the new structure. I'll let @sfshaza2 word smith my prose =)

#### Linux

The executable can be found in your project under `build/linux/<build mode>/bundle/`. Alongside your 

> executable binary in the `bundle` directory there are two directories:
>  - `lib` which contains required `.so` library files, and 
>  - `data` which contains the data assets that your application relies on.

Reviewer 2: ```suggestion
executable binary in the `bundle` directory there are two directories:
 * `lib` contains the required `.so` library files
 * `data` contains the application's data assets, such as fonts or images
```

Author: Yup, much betterer
